src/sentry/api/serializers/models/sentry_app.py

```diff
@@ -4,11 +4,12 @@
 from sentry.auth.superuser import is_active_superuser
 from sentry.api.serializers import Serializer, register
 from sentry.models import SentryApp
+from sentry.models.sentryapp import MASKED_VALUE
 
 
 @register(SentryApp)
 class SentryAppSerializer(Serializer):
-    def serialize(self, obj, attrs, user):
+    def serialize(self, obj, attrs, user, access):
```

Comment:
I don't really like making the signature for `serialize` inconsistent. Our usual pattern for serializers is to use 
```
from sentry.app import env
env.request
```

This is always set by `SentryEnvMiddleware`, so should be safe to use.

I'm +1 on just explicitly passing either `access` or `request` at some point, but I think we need to refactor all of the serializers to also accept that param in that case.

Reply:
was def looking at https://github.com/getsentry/sentry/blob/master/src/sentry/api/serializers/models/organization.py#L125 when I did this - but happy to change to `env.request`

Reply:
Nm, after our discussion looks like access isn't available on `env.request`